base58/__init__.py

```diff
@@ -23,6 +24,16 @@
 alphabet = BITCOIN_ALPHABET
 
 
+def fix_position(position: int) -> int:
+    return position if position >= 0 else 255
```

Comment:
This could be written as `position % 256` and be inlined which I think makes as it's about fitting the number into a byte.